src/sphinx/main.rst

```diff
@@ -48,6 +48,28 @@ Simply add to the site mappings. In your ``build.sbt`` file:
 
     site.siteMappings <++= Seq(file1 -> "location.html", file2 -> "image.png")
 
+
+Preprocessing with Variable Substitution
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The site plugin supports basic variable substitution when copying files from ``src/preprocess``.
```

Comment:
note sure I like `src/preprocess`, maybe `src/site-preprocess` is slightly better, since there may be other preprocessing steps?